cluster/sbin/snappy-nodes.sh

```diff
@@ -448,4 +451,18 @@ else
   fi
   execute "$@"
 fi
-wait
+
+if [ "$isServerStart" ]; then
+  # server status file
+  SERVERS_STATUS_FILE="$SNAPPY_HOME/work/serversstatus"
+  if [ -f $SERVERS_STATUS_FILE ]; then
+    rm $SERVERS_STATUS_FILE
+  fi
+  touch $SERVERS_STATUS_FILE
+  for pid in "${childPids[@]}"; do
+    wait $pid
+    echo $? >> $SERVERS_STATUS_FILE
```

Comment:
For user readability will be good to also add the host in this file

Reply:
ok.